bottles/frontend/ui/details-preferences.blp

```diff
@@ -68,38 +106,17 @@ template DetailsPreferences : .AdwPreferencesPage {
       model:
       StringList {
         strings [
+          _("Disabled"),
           _("Ultra Quality"),
           _("Quality"),
           _("Balanced"),
           _("Performance"),
         ]
```

Comment:
There are [5 levels](https://support.system76.com/articles/use-amd-fsr/#resolution-and-quality) for FSR, but there are only 3 here. I added "Ultra Performance", but we'll need another one to increment. What do you suggest?

```suggestion
        strings [
          _("Disabled"),
          _("Ultra Quality"),
          _("Quality"),
          _("Balanced"),
          _("Performance"),
          _("Ultra Performance"),
        ]
```

Reply:
Out of scope for this specifically :)